Makefile

```diff
@@ -22,10 +22,10 @@ BINARY_EXT_linux :=
 BINARY_EXT_darwin :=
 BINARY_EXT_windows := .exe
 DOCKER_IMAGE_RPM := alectolytic/rpmbuilder
-DOCKER_IMAGE_DEB := appsody/debian-builder
+DOCKER_IMAGE_DEB := appsody/debian-builder:0.1.0
 GH_ORG ?= appsody
-CONTROLLER_VERSION ?=0.2.6
-CONTROLLER_BASE_URL := https://github.com/${GH_ORG}/controller/releases/download/$(CONTROLLER_VERSION)
+CONTROLLER_VERSION ?=0.3.0
```

Comment:
I think you should export this and make it APPSODY_CONTROLLER_VERSION.
That way `make test` will use it instead of `latest`